src/app/content/components/Sidebar.tsx

```diff
@@ -160,21 +172,150 @@ const NavItemComponent: ComponentType<{active?: boolean, className?: string}> =
   >{children}</li>
 );
 
+// tslint:disable-next-line:variable-name
+const ContentLink = styled(ContentLinkComponent)`
+  ${textStyle}
+  display: flex;
+  margin-left: ${iconPadding + iconSize}rem;
+  text-decoration: none;
+
+  li[aria-label="Current Page"] & {
+    font-weight: bold;
+  }
+
+`;
+
 // tslint:disable-next-line:variable-name
 const NavItem = styled(NavItemComponent)`
+  ${textStyle}
   list-style: none;
   font-size: 1.4rem;
-  ${(props) => props.active && css`
-    overflow: visible;
-    position: relative;
+  line-height: 1.6rem;
+  overflow: visible;
+  margin: 0 0 1.5rem 0;
+
+  ${SummaryTitle}, ${ContentLink} {
+    :hover {
+      color: ${theme.textColors.black}
```

Comment:
can this style go on the summaryTitle and ContentLink components? should they share some text styles that are used for the whole sidebar?

Reply:
besides the :hover, what styles you think they could share?